### PR TITLE
Add Assignees automatically when someone creates PR (#667)

### PR DESCRIPTION
**Pull request by gimwonbae**
### Description
Add Assignees automatically when someone creates PR using github action

### Issue
Resolve https://github.com/gluesql/gluesql/issues/667

### CODE REVIEW

**Comment by 24seconds** [comment]
Let me review this pr

File: .github/workflows/assign-pr.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    branches: [ "main" ]
+
+jobs:
+  assign-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Add assignees to an PR
+        run: |
+          HTTP_RESP_CODE=$(curl -o /dev/null  -w "%{http_code}" -X POST -H "Accept: application/vnd.github+json" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.number }}/assignees -d '{"assignees":["${{ github.event.sender.login }}"]}')
```
Comment (24seconds) [inline .github/workflows/assign-pr.yml:14]: [Suggestion and IMO]
It seems this code is directly calling the github API. For the maintenance and sustainability, how about using a github action which is released in market place?  
For example, [Assign-author](https://github.com/marketplace/actions/assign-author) or [Auto author assign](https://github.com/marketplace/actions/auto-author-assign)? It seems it is easy to use and provides a bit more features.

**Review by 24seconds — COMMENTED** [review]
Thank you for the work. I left a comment. Could you take a look?

File: .github/workflows/assign-pr.yml (hunk 1 above)
Comment (gimwonbae) [reply .github/workflows/assign-pr.yml:14]: For complex or multi-step tasks like docker build/push, I agree that you're better off using marketplace actions as you said.

However, I'm not sure what the maintenance advantage is of using an external action for a simple task that can be executed with one simple api call.

Using the Github API is as scalable and easy as using an external action in the marketplace, and it seems more maintainable than having a dependency on an external repo with fewer stars and less reliability.

Please let me know if there are any advantages to using external actions that I've missed 😀.

Comment (24seconds) [reply .github/workflows/assign-pr.yml:14]: Thank you for the opinion.  

I think using existing solution - github action I mean - has two advantages.  

1. We don't need to know about the details.
By using github action, we don't need to care much about small changes. I believe they - github action owner - will follow up the problems, bugs or some changes on behalf of us. (For example, github API's behavior change) 
What we need to do is just updating the github action version. But this is based on the assumption that the owner would maintain the project consistently. 😅 

2. We don't need to care about shell script proficiency.
Maybe it is my bias but I think nowadays people are not so much familiar with shell script code. So it would be a bit hard to maintain the job because reviewer and pr author should know about it. So.. I think it's good to use github action. Well, in this case because it is so simple this is not a big problem. 😄 

To sum up, we agreed the problem is so simple. But the action was different.
- Because it's so simple it's okay to use github action
- Because it's so simple it's okay to use simple script
Therefore I think it's okay to choose either way. So it's own this repo owner's choice. Thank you for the comment again!

cc: @panarch

Comment (gimwonbae) [reply .github/workflows/assign-pr.yml:14]: Thanks for your comments!

p.s. REST API has backward compatibility, so even if the Github API version is changed, there seems to be no side effect.

Comment (24seconds) [reply .github/workflows/assign-pr.yml:14]: @panarch could you check this pr?

Comment (ever0de) [reply .github/workflows/assign-pr.yml:14]: I'm not a maintainer, but I leave comments.
1. We may need scalability. At this time, I think it can be easily added using the functions of github-action. (For example, designate a reviewer)
2. If a REST call fails, functions such as retry attempts will come more intuitively when using a github action rather than writing a shell script.
